bin/improver-nowcast-extrapolate

```diff
@@ -87,9 +87,12 @@ def main():
 
     # extrapolate input data to required lead times
     for i, lead_time in enumerate(lead_times):
-        # cast to float as datetime.timedelta cannot accept np.int
-        timestep = datetime.timedelta(seconds=60.*lead_time)
-        forecast_cube = advection_plugin.process(input_cube, timestep)
+        if i == 0:
+            forecast_cube = input_cube
+        else:
+            # cast to float as datetime.timedelta cannot accept np.int
+            timestep = datetime.timedelta(seconds=60.*lead_time)
+            forecast_cube = advection_plugin.process(input_cube, timestep)
```

Comment:
I thought about that. Hopefully - but it's probably doing extra processing that we may/may not want. It depends whether we'd rather pass it through the plugin for consistency, or do as little work as possible so that we can finish earlier.
I can give it a go and report timings?

Reply:
Done.